frontend/packages/pipelines-plugin/src/utils/__tests__/pipeline-test-data.ts

```diff
@@ -290,3 +290,59 @@ export const mockRunDurationTest = [
   pipelineTestData[PipelineExampleNames.WORKSPACE_PIPELINE].pipelineRuns[DataState.SUCCESS],
   pipelineTestData[PipelineExampleNames.CLUSTER_PIPELINE].pipelineRuns[DataState.SUCCESS],
 ];
+
+export const taskRunWithResults: TaskRunKind = {
+  apiVersion: 'tekton.dev/v1beta1',
+  kind: 'TaskRun',
```

Comment:
Please move this test data to `taskrun-test-data.ts` (This file contains all the taskrun test data).

Reply:
moved